tests/connection/connection.cpp

```diff
@@ -94,7 +94,9 @@ TEST(Connection, componentlessVariableInvalidConnection) {
     comp1->addVariable(v1);
     m.addComponent(comp1);
     libcellml::Variable::addEquivalence(v1, v2);
-    std::string a = m.serialise(libcellml::Format::XML);
+
+    libcellml::Printer printer(libcellml::Format::XML);
+    const std::string a = printer.printModel(m);
```

Comment:
I just added the const to make the code faster.  I have only added const where the string is not changed further thus some are const and some are not.